### DISCUSSION TITLE
是否能够设置缓存的位置

### DISCUSSION THREAD

**Discussion by zcy85611** [messense/aliyundrive-webdav#378] [Q&A] [answered]
### 问题描述

由于路由器是nand flash，反复擦写容易出问题，想把缓存位置设置到外置的U盘上，避免过度擦写，但是在luci界面里没有找到具体的选项，想问下是否支持设置缓存位置？如不支持，后续版本是否考虑加上？感谢解答！

### 重现步骤

无

### 版本

1.3.1

### 运行平台

openwrt

### 日志

无

**Comment by messense** [comment, marked as answer]
不需要，缓存是内存。
